ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable-4.15-upgrade-from-stable-4.13.yaml

```diff
@@ -0,0 +1,97 @@
+base_images:
+  ansible:
+    name: "4.15"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.15"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  hypershift-operator:
+    name: "4.15"
+    namespace: ocp
+    tag: hypershift-operator
+  openstack-installer:
+    name: "4.15"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.15"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.14"
+  tools:
+    name: "4.15"
+    namespace: ocp
+    tag: tools
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  intermediate414:
+    release:
+      architecture: amd64
+      channel: candidate
+      version: "4.14"
+  latest:
+    release:
+      architecture: amd64
+      channel: fast
+      version: "4.13"
+  target:
+    release:
+      architecture: amd64
+      channel: candidate
+      version: "4.15"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: gcp-ipi-baselinecaps-none-f14
+  cron: 12 22 13,27 * *
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      BASELINE_CAPABILITY_SET: None
+      PERSISTENT_MONITORING: "false"
+      UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.14 and @gcp-ipi and not @storage
+        and not @workloads and not @console and not @fips'
```

Comment:
There is no need to exclude workloads, right? and we need `@network-ovnkubernetes`.

Reply:
Confirmed with @kasturinarra , we don't need to run them.

Reply:
Fine. Please append @network-ovnkubernetes

Reply:
`BASELINE_CAPABILITY_SET: None` only disables the optional components available in 4.13 (the install version). It includes "baremetal marketplace openshift-samples Console Insights Storage CSISnapshot NodeTuning". I don't see workload will be impacted.

Reply:
Not really. The test is installing 4.13 with None capset where build and dc is enabled by default, then upgrading to 4.14, preparing data by executing test-cases, then upgrading to 4.15. Based on the logic that an enabled cap can't be disabled any more, I think build and dc will be always present in the cluster.

Reply:
@shellyyang1989 got it make sense now, thank you !!